libraries/app/database_api.cpp

```diff
@@ -1554,10 +1554,12 @@ vector<market_trade> database_api_impl::get_trade_history( const string& base,
          {
             trade.sequence = -itr->key.sequence;
             trade.side1_account_id = itr->op.account_id;
+            trade.side1_was_selling = itr->op.receives;
```

Comment:
It looks strange. "Was selling" something but "received" it?

Another issue is the data type of the new fields. IMO it should be string, and the value should be the `const string& base` or `const string& quote`.

Reply:
What do you think about `side1_received` and `side2_received` as the names?

I am not sure if i understand the suggested type, you mean something like:
```
BOOST_CHECK_EQUAL( "20000 CNY", history[0].side1_was_selling.amount.value );
```
?

Reply:
If you add comments/docs to every field of `struct market_trade`, and/or check the returned data, I guess you'll know what data we're returning and what's missing. The purpose of the API is to return useful clear human-readable data. We don't need to return redundant data.

Reply:
Ok, good luck then. thank you.

Reply:
I meant to return something like "side1_was_selling": "CNY".